### ISSUE TITLE
Add particle to the metadata consistent with the particle class

### ISSUE THREAD

**Issue by ahashibon** [simphony/simphony-metadata#43]
the metadata should contain all basic data structures, including particle: 

```
PARTICLE:

 definition: a particle model entity

  parent: CUBA.CUDS_COMPONENT

  CUBA.POSITION: 

    default: [0 0 0 ]

```

The proposal is also to change coordinate to position attribute. Since PARTICLE is a cuds component it will have id and data container.

**Comment by mehdisadeghi** [comment]
Shall we consider adding Mesh and Lattice as well?

**Comment by SGGgarcia** [comment]
If this is ok, should we do the same with the bonds?

**Comment by stefanoborini** [comment]
Fixed by #50
